packages/enzyme-test-suite/test/wrapperSandbox-spec.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { ReactWrapper, configure } from 'enzyme';
+import { get } from 'enzyme/build/configuration';
+
+import {
+  trackWrapper,
+  unmountAllWrappers,
+} from 'enzyme/build/wrapperSandbox';
+
+const originalConfig = get();
```

Comment:
this should probably be populated `before` or `beforeEach` rather than at require time